src/ert/_c_wrappers/job_queue/job_queue_node.py

```diff
@@ -170,13 +171,26 @@ def _job_monitor(self, driver, pool_sema, max_submit):
 
         current_status = self.refresh_status(driver)
 
+        initial_sleep_seconds = 1
+        time_sleep_seconds = initial_sleep_seconds
+        max_sleep_seconds = 30
```

Comment:
Just wondering if we should decrease it to 15 secs.

Reply:
we can, no problem

Reply:
done.